src/components/Composer.vue

```diff
@@ -653,6 +653,18 @@ export default {
 					})
 					this.state = STATES.ERROR
 				})
+
+			// Sync sent mailbox when it's currently open
+			const account = this.$store.getters.getAccount(this.selectedAlias.id)
+			if (this.$route.params.mailboxId === account.sentMailboxId) {
+				return setTimeout(() => {
```

Comment:
don't return the timer. if the method returns anything then it should be a promise